cmake/TenzirRegisterPlugin.cmake

```diff
@@ -198,6 +198,22 @@ function (TenzirCompileFlatBuffers)
     INTERFACE $<BUILD_INTERFACE:${output_prefix}>
               $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
 
+  if ("${CMAKE_PROJECT_NAME}" STREQUAL "Tenzir")
```

Comment:
If you want to test this working, just add `include "data.fbs";` at the top of any `*.fbs` file in any closed-source plugin. That used to fail.